Apps/Src/Voltage.c

```diff
@@ -99,7 +99,14 @@ SafetyStatus Voltage_CheckStatus(void){
  */
 SafetyStatus *Voltage_GetModulesInDanger(void){
 	static SafetyStatus checks[NUM_BATTERY_MODULES];
-	uint32_t open_wires = Voltage_GetOpenWire();
+	Voltage_GetOpenWire();
+	uint32_t open_wires = 0;
+
+	//put all the bits from each minion's system_open_wire variable into one variable
+	for(int k = 0; k < NUM_MINIONS; k++){
+		open_wires += Minions[k].system_open_wire & 0x0FF;	//there are at most 8 modules per IC 
+		open_wires = open_wires << 8;
```

Comment:
Since it's not a nice power of 2 number (should be 9), may be better to just have an array instead of an integer bitmap.